### ISSUE TITLE
Containers not restarting after reboot

### ISSUE THREAD

**Issue by markbeierl** [canonical/mysql-k8s-operator#317]
## Steps to reproduce

1. Install Openstack with a database per service (default)
2. After a couple of days and VMs created/deleted, cleanly shut down and reboot the server
3. View juju status after reboot

## Expected behavior

All services restart after reboot

## Actual behavior

```
cinder-mysql/0*              blocked   idle   10.1.182.215         failed to recover cluster.
glance-mysql/0*              blocked   idle   10.1.182.205         failed to recover cluster.
horizon-mysql/0*             blocked   idle   10.1.182.240         failed to recover cluster.
keystone-mysql/0*            blocked   idle   10.1.182.248         failed to recover cluster.
neutron-mysql/0*             blocked   idle   10.1.182.243         failed to recover cluster.
nova-mysql/0*                blocked   idle   10.1.182.227         failed to recover cluster.
```

## Versions

Operating system: Ubuntu 22.04.3 LTS
Juju CLI: 3.2.3-genericlinux-amd64
Juju agent: 3.2.0
Charm revision: 8.0/candidate
microk8s: MicroK8s v1.26.8 revision 5883

## Log output
- [juju-debug.log.gz](https://github.com/canonical/mysql-k8s-operator/files/12860190/juju-debug.log.gz)
- [mysql-error.log](https://github.com/canonical/mysql-k8s-operator/files/12860189/mysql-error.log)

**Comment by markbeierl** [comment]
New [juju-debug.log.gz](https://github.com/canonical/mysql-k8s-operator/files/12861464/juju-debug.log.gz) with bootstrap and reboot all done within 2 hours.

**Comment by paulomach** [comment]
Hi @markbeierl , I believe the fix is on the way on PR #324

**Comment by markbeierl** [comment]
[logs.GZ](https://github.com/canonical/mysql-k8s-operator/files/12918194/logs.GZ)
This is actually a .tgz, but GitHub refused to accept a file that ends in .tgz or .TGZ.

It contains:
- juju debug-log for the openstack model from creation until time of reboot
- mysql /var/log/mysql/error.log from install to time of reboot
- juju debug-log --replay from after reboot
- mysql /var/log/mysql/error.log from after reboot
Please let me know if there are any other logs I can provide
